sql/6_1_0-to-7_0_0_upgrade.sql

```diff
@@ -274,3 +274,19 @@ ALTER TABLE `ccda` ADD COLUMN `transaction_id` BIGINT(20) COMMENT 'fk to transac
 #IfMissingColumn form_care_plan date_end
 ALTER TABLE `form_care_plan` ADD `date_end` DATETIME DEFAULT NULL, ADD `reason_code` VARCHAR(31) DEFAULT NULL, ADD `reason_description` TEXT, ADD `reason_date_low` DATETIME DEFAULT NULL, ADD `reason_date_high` DATETIME DEFAULT NULL;
 #EndIf
+
+#IfNotRow3D layout_options form_id DEM field_id street edit_options U
```

Comment:
check out @sjpadgett's cool function to add options to fields:
https://github.com/openemr/openemr/blob/master/sql/6_1_0-to-7_0_0_upgrade.sql#L104